### ISSUE TITLE
Can we use this code online?

### ISSUE THREAD

**Issue by RuyiLi** [rauchg/chat-example#21]
If I were to make a website with all these, would it work?

**Comment by staeff** [comment]
Yes, the only requirement ist Node.js. With the following steps I could run this example:

    git clone https://github.com/rauchg/chat-example.git
    cd chat-example/
    npm install
    node index.js

See also https://github.com/rauchg/chat-example/pull/5, where someone has created a version of the example that runs on Heroku
